acq4/filetypes/Analyze75.py

```diff
@@ -231,11 +220,11 @@ def parseNii(headerFH, imgFile):
             data = headerFH.read(size)
         elif m.magic == 'nii\0':               ## data is in a separate .img file
             imgFile = os.path.splitext(imgFile)[0] + '.img'
-            fh = open(imgFile, 'rb')
-            fh.seek(m.vox_offset)
-            data = fh.read(size)
+            with open(imgFile, 'r', encoding="utf-8") as fh:
```

Comment:
Sure about this change? We don't have a test for it..

Reply:
`np.fromstring` wants strings, not bytes.

Reply:
But `data` is likely binary, not unicode. That means opening as a unicode string will likely just fail. I think the correct fix is to use `np.frombuffer` anywhere we were previously using `fromstring`.